app/assets/javascripts/discourse/widgets/post-cooked.js.es6

```diff
@@ -161,7 +161,13 @@ export default class PostCooked {
 
       ajax(`/posts/by_number/${topicId}/${postId}`)
         .then(result => {
+          const post = this.decoratorHelper.getModel();
+          const quotedPosts = post.quoted || {};
+          quotedPosts[result.id] = result;
+          post.set("quoted", quotedPosts);
+
           const div = $("<div class='expanded-quote'></div>");
+          div.data("post-id", result.id);
```

Comment:
all of this can be easily replaced by vanilla js:

```
const div = document.createElement("div");
div.classList.add("expanded-quote");
div.setAttribute("data-post-id", result.id);
div.innerHTML =  result.cooked;
```

You will have to $(div) in forEach loop though, so it's only a partial step for now.

Reply:
The `div` object is also used in multiple places below which make it hard to use vanilla JavaScript.